File: viajes365-backend/Viajes365RestApi/Entities/Comment.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Viajes365RestApi.Entities
{
    public class Comment : Base
    {
        [Key]
        public long CommentId { get; set; }
        [Required]
        [StringLength(255)]
        public string Body { get; set; }
        
        public long TopicId { get; set; }
        public virtual Topic Topic { get; set; }
        
        public long UserId { get; set; }
        public virtual User User { get; set; }
    }
}

File: viajes365-frontend/src/app/_models/comment.ts
import { Topic } from './topic';
import { User } from './user';

export class Comment {
  commentId!: number;
  body!: string;
  status!: string;
  topicId!: number;
  topic!: Topic;
  userId!: number;
  user!: User;
  isDeleting = false;
  created!: Date;
  active!: boolean;
}
